corehq/apps/app_manager/util.py

```diff
@@ -572,7 +572,12 @@ def get_sort_and_sort_only_columns(detail_columns, sort_elements):
         if match:
             del sort_elements[field]
             column_index = int(match.group(1))
-            column = detail_columns[column_index]
+            try:
```

Comment:
These exceptions generally shouldn't happen, right? If they do happen, does the error text show to the user when they make a build, or does it only show up in sentry?

Reply:
They result in a generic build error ("An error occurred. Reload the page and click Make New Version to try again") and an error in Sentry.

Reply:
Okay, cool. I do like the custom text, especially for non-devs looking at sentry.